_src/app/resources/App.styl

```diff
@@ -0,0 +1,158 @@
+            margin 5px 0 0 5px
+            font-size 40px
+            letter-spacing -2px
+        .version
+            font-size 10px
+            text-decoration none
+            vertical-align sub
+            margin-left -10px
+            letter-spacing 0
+        .version:hover
+            text-decoration underline !important
+        >img
+            position absolute
+            top -3px
+            right 0
+    [data-dojo-attach-point='popupBlockerBlurb']
+        display none
+
+    /* SIDE BAR */
+    .side-bar-container
+        overflow-y auto
+        overflow-x hidden
+        position absolute
+        left $container-margin
+        top 75px
+        bottom $container-margin
+        &.open
+            width $side-bar-width
+            animation-name sidebar-open
+            animation-duration $animation-duration
+        &.closed
+            width 0
+            border none
+            animation-name sidebar-close
+            animation-duration $animation-duration
+        .sherlock
+            position inherit
+            width 100%
+            padding 0
+            .input-group
+                width 100%
+        .side-bar-padder
+            padding $side-bar-padding
+            width $side-bar-width
+            margin -1px
+        .esri-print
+            width 100%
+            padding 0
+            header
+                visibility hidden
+                display none
+        .panel-body
+            padding 8px
+
+    /* CENTER (MAP) */
+    .center-container
+        position absolute
+        top 75px
+        bottom $container-margin
+        right $container-margin
+        overflow hidden
+        .sidebar-toggler
+            position absolute
+            left 0
+            top 0
+            z-index 99
+            margin -2px
+        &.sidebar-open
+            left $side-bar-width + (2 * $container-margin)
+            animation-name center-sidebar-open
+            animation-duration $animation-duration
+        &.sidebar-closed
+            left $container-margin
+            animation-name center-sidebar-close
+            animation-duration $animation-duration
+        [data-dojo-attach-point='mapDiv']
+            position absolute
+            top 0
+            bottom 0
+            left 0
+            right 0
+            .esri-popup__main-container
+                width 290px
+
+@keyframes sidebar-open
+    from
+        width 0
+    to
+        width $side-bar-width
+@keyframes sidebar-close
+    from
+        width $side-bar-width
+    to
+        width 0
+@keyframes center-sidebar-close
+    from
+        left $side-bar-width + (2 * $container-margin)
+    to
+        left $container-margin
+@keyframes center-sidebar-open
+    from
+        left 0
+    to
+        left $side-bar-width + (2 * $container-margin)
+
+.matches-table
+    width 280px
+
+/* GENERAL STUFF */
+/* remove orange outline in chrome */
+*focus
+    outline none
+
+.map .tooltip
```

Comment:
Not sure since this project doesn't use tooltips. I put it in Atlas because it was a pain figuring this out. It resolves a collision between the `tooltip` CSS class in Esri and Bootstrap. I'll remove it for now since it's likely different at 4.x.

f074a06